### ISSUE TITLE
Importing SH collection - Getting logged out on sending fee entry

### ISSUE THREAD

**Issue by Lycan2022** [ergo-auction-house/frontend#72]
I tried to import my collection from skyharbor. Everything goes smoothly until I want to pay the fee. After entering the sending password of my wallet, AH disconnects my wallet, and I get returned to the homepage logged out.

**Comment by anon-real** [comment]
Do you still get this error?

**Comment by Lycan2022** [comment]
Yeah I think it has to with me trying to link same twitter account to multiple wallets. 

Same issue: https://github.com/ergo-auction-house/frontend/issues/76

**Comment by Lycan2022** [comment]
Problem #76, but not seem to be able to import a SH collection. It still kicks me out when I want to pay the fee. I did already dis-/reconnect the wallet. However it did not seem to work.

**Comment by Lycan2022** [comment]
Addition to the last post, providing screenshots:
1. ![image](https://github.com/ergo-auction-house/frontend/assets/120051677/6dfe7a8a-d66a-4bac-b6aa-1ede309bf507)
Just after entering the password in nautilus to pay the fee. I had to do a screenrecording to capture it, because it disappears very quickly. Followed by the screen below (logged out and entries lost).

2. ![image](https://github.com/ergo-auction-house/frontend/assets/120051677/41ec1e93-5eb9-495d-aa74-ee7662f6bec1)
